.github/workflows/tests.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       - name: Run build:ts
         run: yarn nx run-many --target=build:ts --nx-ignore-cycles --skip-nx-cache
       - name: Run tests
-        run: yarn nx affected --target=test:unit --nx-ignore-cycles
+        run: yarn nx affected --target=test:unit --nx-ignore-cycles -- --runInBand
```

Comment:
Mmm. I'd say could not measure difference by moving to full swc + updated versions as tested locally. 

But if recommend to test locally with the command provided above from master, this is much easier (if you're on Linux). 

The runInBand who be faster equally. 

If you really want I can restart another with just runInBand  PR but I'm pretty confident on this one.

Reply:
> I think even if it is a one-line change it will take longer to discuss than the dependency update.

Yes. The doc is taking most of the time. I'll try to provide a better way to document outside of the PR... otherwise I feel we could loose context

Reply:
@gu-stav here it is: https://github.com/strapi/strapi/pull/16743

Reply:
![image](https://github.com/strapi/strapi/assets/259798/737e0876-5adb-44ac-bd6e-e4524b43fde7)

Reply:
Thank you. Could you please remove it from this PR? I think we can merge it then and move on to your next ideas.

Reply:
Done :)